src/System.Security.AccessControl/ref/Configurations.props

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      netcoreapp;
```

Comment:
I don't think this ref needs to have netcoreapp and uap added. You're not doing anything special for that ref. Leave it as it was.